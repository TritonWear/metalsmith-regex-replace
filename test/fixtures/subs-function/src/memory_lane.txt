One for the money
Two for pussy and foreign cars
Three for Aliz√©, niggas deceased or behind bars