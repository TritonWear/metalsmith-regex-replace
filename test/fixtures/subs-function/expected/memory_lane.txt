One for the money
Two for women and foreign cars
Three for Aliz√©, brethren deceased or behind bars